src/main/java/Task.java

```diff
@@ -0,0 +1,41 @@
+public class Task {
+    private String item;
+    private int index;
+    private String status;
+    private String type;
```

Comment:
I think the namings are not entirely clear, for example can the "type" be named as "classType" instead?

Reply:
That makes more sense, I will edit this. Appreciate it!